frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -864,18 +867,20 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
                 {filteredNamespaces.map((ns, i) => {
                   const isLongNs = ns.name.length > NS_LONG;
                   return (
-                    <GridItem sm={(ns.name === serverConfig.istioNamespace && this.state.displayMode === OverviewDisplayMode.EXPAND) ? lg : sm}
```

Comment:
Minor: it looks like the controlplane card's height doesn't match the other cards now with the istio api disabled:
![Screenshot from 2023-01-09 12-41-30](https://user-images.githubusercontent.com/6226732/211372808-ba082081-ecce-4d33-bbbd-3cb777e17183.png)